ui/narrative/methods/run_metaSPAdes/spec.json

```diff
@@ -58,7 +58,8 @@
                 },
                 {
                     "input_parameter": "read_libraries",
-                    "target_property": "read_libraries"
+                    "target_property": "read_libraries",
+                    "target_type_transform": "list<ref>"
```

Comment:
Is there a check such that if I try to submit multiple files to a metaSPAdes job by directly contacting NJSW (and thus bypass the narrative) an error is thrown?